### PR TITLE
[MIG] Initialice branch for v16

### PR DESCRIPTION
**Pull request by isaako34**
- CI files are initialized (variables.sh, gitignore, README.md, CONTRIBUTING.md, .travis.yml)

- Modules have not been migrated yet.

### CODE REVIEW

**Comment by isaako34** [comment]
Hello @hugho-ad could you review it, please.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  - cp test-oca_dependencies.txt oca_dependencies.txt   
+  - travis_install_nightly
+
+script:
+  - source ${TRAVIS_BUILD_DIR}/variables.sh
+  - travis_run_tests
+
+after_success:
+  - source ${TRAVIS_BUILD_DIR}/variables.sh
+  - travis_after_tests_success
```
Comment (hugho-ad) [inline .travis.yml:39]: EOF

File: variables.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export BASE_IMAGE="vauxoo/odoo-160-image"
+export DOCKER_IMAGE_REPO="quay.io/vauxoo/addons_vauxoo"
+export PSQL_VERSION="11"
```
Comment (hugho-ad) [inline variables.sh:3]: i think you can use 13.0

**Review by luisg123v — CHANGES_REQUESTED** [review]
I think we need to close this PR, I first need to apply changes in v15 to use VXCI instead of MQT.

**Comment by luisg123v** [comment]
Superseded by #1600

**Pull request closed without merging**